### PR TITLE
docs: fix introduction markup

### PR DESCRIPTION
**Pull request by jrudolph**

### CODE REVIEW

File: docs/src/main/paradox/introduction.md [hunk 1]
```diff
@@ -49,15 +49,17 @@ Alternatively, you can bootstrap a new project with Akka HTTP already configured
 
 @@@ div { .group-scala }
 For Scala (sbt)
-:  ```sh
-    sbt new akka/akka-http-quickstart-scala.g8
-    ```
+:
```
Comment (jrudolph) [inline docs/src/main/paradox/introduction.md:52]: The previous style broke markup further down. Now it was replaced by significant white space at the end of the line...

Comment (ennru) [reply docs/src/main/paradox/introduction.md:52]: In other places we've added `&#9;` to make the whitespace stick.

Comment (jrudolph) [reply docs/src/main/paradox/introduction.md:52]: It's not about whitespace in the generated file but about the markdown syntax to use two trailing spaces to continue a line. So, not sure how that could help here.

Comment (jrudolph) [reply docs/src/main/paradox/introduction.md:52]: Hmm, that doesn't seem to be all that's needed to fix it. I also have to revert ee0b09d7c3534eb170466454341db7107b91889a to fix the issue.

Comment (jrudolph) [reply docs/src/main/paradox/introduction.md]: Nothing to with that after all. It was just a missing newline :roll_eyes: Sorry for the noise.

**Review by ennru — APPROVED** [review]
Great change!

**Review by raboof — APPROVED** [review]

**Pull request merged**